badges/badge.css

```diff
@@ -0,0 +1,178 @@
+@import url(/CSS/nav.css);
+
+
+:root {
+
+    --green-badge: #16a34a;
+    --red-badge: #dc2626;
+    --container-color: #f9fafb;
+    --box-shadow: 10px 10px 8px #e5e7eb;
+}
+
+
+.main-container {
+
+    display: flex;
+    flex-direction: column;
+    max-width: 82%;
+    margin-top: 2rem;
+    position: relative;
+    padding-left: 19rem;
+}
+
+h3 {
+
+    margin-bottom: 0.5rem;
+    font-size: 1.8rem;
+}
+
+.badge-heading {
+
+    margin-top: 1rem;
+    font-weight: bold;
+    font-size: 1.3rem;
+    margin-bottom: 1rem;
+}
+
+hr {
+    margin-top: 0.8rem;
+}
+
+.icon-badges {
+
+
+    display: flex;
+    flex-direction: column;
+    max-width: 82%;
+    position: relative;
+    padding-left: 19rem;
+
+}
+
+.icons-container {
+
+    display: flex;
+    justify-content: center;
+    border: 1px solid;
+    padding: 3rem;
+    border-radius: 0.5rem;
+    background-color: var(--container-color);
+    border: none;
+    box-shadow: var(--box-shadow);
+    margin-bottom: 1rem;
+}
+
+.icon {
+
+    font-size: 2rem;
+    margin: 0rem 3rem;
+}
+
+.notif-cart {
+
+    position: absolute;
+    bottom: 84rem;
+    left: 36.7rem;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+    color: var(--white);
+    font-weight: bold;
+    padding: 0.1rem 0.4rem;
+    font-size: small;
+}
+
+.notif-whatsapp {
+
```

Comment:
lots of stuff are being repeated in  .notif-cart,notif-whatsapp,.notif-mail
better make a different class which you can use side by side with these classes